korman/nodes/node_python.py

```diff
@@ -370,6 +425,31 @@ def update(self):
             self.inited = True
 
 
+class PlasmaAttribDropDownListNode(PlasmaAttribNodeBase, bpy.types.Node):
+    bl_category = "PYTHON"
+    bl_idname = "PlasmaAttribDropDownListNode"
+    bl_label = "Drop Down List Attribute"
+
+    pl_attrib = "ptAttribDropDownList"
+
+    def _list_items(self, context):
+        items = []
+        attrib = self.to_socket
+        if attrib is not None:
+            items = [(option.value, option.value, "") for option in attrib.attribute_arguments.options]
+        return items
```

Comment:
It might be better to:
```python
if attrib is not None:
     return [(option.value, option.value, "") for option in attrib.attribute_arguments.options]
else:
    return []
```